src/access/moderator-access.js

```diff
@@ -0,0 +1,65 @@
+// 'use strict'
+const AccessController = require('orbit-db-access-controllers/src/access-controller-interface')
```

Comment:
yeah probably, there is isnt much in the access controller interface, wanted to try the accesscontroller again and modify the build, but this might also be easier

Reply:
end up just using event emitter as mentioned